inc/notificationeventmailing.class.php

```diff
@@ -137,19 +137,23 @@ static public function send(array $data) {
          }
          $mmail->Subject  = $current->fields['name'];
 
+         // manage item attached documents
+         $document_items = $DB->request('glpi_documents_items', [
```

Comment:
I guess this should be conditionned on `$CFG_GLPI['attach_ticket_documents_to_mail']` to prevent useless queries when not needed

Reply:
Done. I also change query to fetch only `documents_id`.